CMakeLists.txt.ros2

```diff
@@ -0,0 +1,146 @@
+cmake_minimum_required(VERSION 3.5)
+project(common_robotics_utilities)
+
+find_package(ament_cmake_ros REQUIRED)
+
+find_package(geometry_msgs REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(sensor_msgs REQUIRED)
```

Comment:
Same as above, as far as I can tell, `std_msgs` and `sensor_msgs` dependencies can be removed now.

Reply:
See https://github.com/calderpg/common_robotics_utilities/pull/6/commits/c10dc233726126147cf820a91d05f7a5c3a32ec2.